cclient: update readme.md (#1373)
iotaledger/entangled: cclient/README.md

## Running test cases

```
$ bazel test //cclient/...
```

> ## Using CClient library in a Bazel project.  

Reviewer: ```suggestion
## Using CClient library in a Bazel project
```

> CClient library is a component in the entangled repository, in order to use it in a Bazle project

Reviewer: ```suggestion
CClient library is a component in the entangled repository, in order to use it in a Bazel project
```

> * create a new Bazle project  

Reviewer: ```suggestion
* create a new Bazel project  
```

* iota_client_traverse_bundle()

## API Documentation  

> The documentation can genetrated by [Doxygen](http://www.doxygen.nl/).  

Reviewer: ```suggestion
The documentation can be generated by [Doxygen](http://www.doxygen.nl/).  
```

```
$ cd entangled
$ doxygen ./Doxyfile
```

> After building the document, you are able to open the document in `entangled/out/html/index.html` via Chrome or Firefox.  

Reviewer: ```suggestion
After building the documentation, you can open it via `entangled/out/html/index.html` on your favorite web browser.  
```
